.gitlab-ci.yml

```diff
@@ -0,0 +1,70 @@
+include:
+  - project: 'Northern.tech/Mender/mendertesting'
+    file: '.gitlab-ci-github-status-updates.yml'
+
+stages:
+  - test
+  - publish
+
+test:unit:
```

Comment:
I am very late here (I was not subscribed to this repo by mistake), but we could have done all this with the `.gitlab-ci-check-golang-unittests.yml` template. What do we have in this file that differs from the template?